addons/settings/BW_Settings.inc.sqf

```diff
@@ -108,11 +108,14 @@ if (["diwako_dui_main"] call ACEFUNC(common,isModLoaded)) then { _settings appen
 ["diwako_dui_indicators_range", 20], // User-Setting
 
 // DUI - Squad Radar - Radar
-["diwako_dui_compassRange", 35, true], // User-Setting
+["diwako_dui_compassRange", 50], // User-Setting (this one sets the starting range)
```

Comment:
because it's a user setting (that isn't forced) this ends up not doing anything
limitation of the bw setting / CBA; the client's setting with have priority

we could one-time manually set it for them?